Consul/Client_Options.cs

```diff
@@ -32,23 +32,29 @@ public class QueryOptions
         public static readonly QueryOptions Default = new QueryOptions()
         {
             Consistency = ConsistencyMode.Default,
+            Namespace = Environment.GetEnvironmentVariable("HTTPNamespaceEnvName"),
             Datacenter = string.Empty,
             Token = string.Empty,
             WaitIndex = 0
         };
 
         /// <summary>
-        /// Providing a datacenter overwrites the DC provided by the Config
+        /// Namespace is the name of the namespace to send along for the request when no other Namespace is present in the QueryOptions.
+        /// </summary>
+        public string Namespace { get; set; }
```

Comment:
I think it is good to mention in a comment that `Namespace` is a consul enterprise feature.